web/src/templates/newsTemplate.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+
+import Layout from '../components/layout';
+import Hero from '../components/hero';
+import NewsContent from '../components/newsContent';
+import RecentNewsList from '../components/recentNewsList';
+
+export default function NewsTemplate({ data, location }) {
+    const {
+        newsItem,
+        news: { edges },
+    } = data;
+
+    const { heroImage, newsTitle } = newsItem;
+
+    const { pathname } = location;
```

Comment:
Do we still need pathname? I don't see where it's being used in [recentNewsList.js](https://github.com/bckrm/airtec/blob/main/web/src/components/recentNewsList.js)